src/test/java/io/cdap/plugin/gcp/publisher/source/PubSubSubscriberTest.java

```diff
@@ -0,0 +1,20 @@
+package io.cdap.plugin.gcp.publisher.source;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+
+import static org.mockito.Mockito.when;
+
+public class PubSubSubscriberTest {
+
+  @Test
```

Comment:
I'm still working on writing these tests. I just want comments on the rest of the code so far.